src/components/expanded-state/custom-gas/FeesPanel.js

```diff
@@ -91,23 +94,58 @@ const GAS_FEE_INCREMENT = 1;
 const MAX_BASE_FEE_RANGE = [1, 3];
 const MINER_TIP_RANGE = [1, 2];
 
+const LOWER_THAN_SUGGESTED = 'Lower than suggested';
+const HIGHER_THAN_NECESSARY = 'Higher than necessary';
+
+const ALERT_MESSAGE_HIGHER_MINER_TIP_NEEDED =
+  'Setting a higher miner tip is recommended to avoid issues.';
+const ALERT_MESSAGE_HIGHER_MAX_BASE_FEE_NEEDED =
+  'Setting a higher max base fee is recommended to avoid issues.';
+const ALERT_MESSAGE_LOWER =
+  'Double check that you entered the correct amount—you’re likely paying more than you need to!';
```

Comment:
are these long dashes part of a spec? look kinda meh imo ?

Reply:
there's no design, this is what's currently on prod but i changed the `gas price` word